Makefile

```
@@ -1,32 +1,54 @@
 #!make
 
-CC ?= clang
+CC    ?= clang
+NPROC := $(shell nproc)
+
+all: clean libyuri cmake binaries
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)